static/src/js/containers/GranuleFiltersContainer/GranuleFiltersContainer.jsx

```diff
@@ -92,7 +92,11 @@ export const GranuleFiltersContainer = (props) => {
 
   const onHandleSubmit = () => {
     // Pass the internal `Formik` values
-    handleSubmit(values)
+    // Waits for the next event cycle to execute
+    // Allows validation for temporal selection to work as expected
+    setTimeout(() => {
```

Comment:
This change scares me. @eudoroolivares2016 you added this correct? Is this to bypass a side-effect of the EDSC-4218 bug?

Reply:
I added this because this [commit](https://github.com/nasa/earthdata-search/commit/0944b7229d46144dd74cda601509337d7d695d4e) introduced an issue where the validation was happening after submission.

This is not a new change. That settimeout was originally on the function for the last 5 years prior to that commit above. I had thought the issue would be fixed by this change and was caused by Vite when I made it but, its coming from this.